README.md

```diff
@@ -43,6 +39,19 @@ The MySQL users and their privileges. A user has the values `name`, `host` (defa
 
 (RedHat/CentOS only) If you have enabled any additional repositories (might I suggest geerlingguy.repo-epel or geerlingguy.repo-remi), those repositories can be listed under this variable (e.g. `remi,epel`). This can be handy, as an example, if you want to install later versions of MySQL.
 
+    mysql_incompatible_packages:
```

Comment:
I'd rather not do this as part of the `geerlingguy.mysql` role; in general, if there are packages that shouldn't be installed on a given server, or other setup tasks outside the purview of MySQL configuration, a `pre_task` or another setup role that runs prior to this role should do it.

As an example, I make sure to always use minimal base boxes for CentOS/RHEL/Debian/Ubuntu, which allow me to build up on top of them only what's needed. If the base image you use with Vagrant, DigitalOcean, AWS, etc. has extra cruft, you need to switch to a different, more minimalist image, or add a task to remove the packages prior to this role.